src/cohortfinder_choosehappy/cohortfinder_colormod_original.py

```diff
@@ -327,10 +398,15 @@ def runCohortFinder(args):
             for fname in fnamessub:
                 print(fname)
                 fullfname = glob.glob(f"{basedir}/**/{fname}*thumb*small*")
+                if (len(fullfname) ==0):
+                    error_message = f"There is no thumbnail images in the subfolders of {basedir}. Please check!"
+                    logging.error(error_message)
+                    raise ValueError(error_message)

```

Comment:
A couple of minor things. 

1. Line 400 should be changed to:

```python
fullfname = glob.glob(f"{basedir}/{fname}/{fname}*thumb*small*")
```
No need for wildcards (**) when we know the directory structure output of histoqc

2. The error message should be more specific:
```python
fullpath = os.path.join(basedir, fname)
error_message = f"No small thumbnail images found in {fullpath}. Please check!"
```